quickwit/quickwit-codegen/src/codegen.rs

```diff
@@ -311,21 +375,32 @@ fn generate_tower_mailbox(mailbox_name: &Ident, error_path: &syn::Path) -> Token
             }
         }
 
+        impl <A: quickwit_actors::Actor> Clone for #mailbox_name<A> {
+            fn clone(&self) -> Self {
+                let inner = MailboxAdapter {
+                    inner: self.inner.clone(),
+                    phantom: std::marker::PhantomData,
+                };
+                Self { inner }
+            }
+        }
+
+        use tower::Service;
+
         impl<A, M, T, E> tower::Service<M> for #mailbox_name<A>
         where
             A: quickwit_actors::Actor + quickwit_actors::Handler<M, Reply = Result<T, E>> + Send + Sync + 'static,
             M: std::fmt::Debug + Send + Sync + 'static,
             T: Send + Sync + 'static,
             E: std::fmt::Debug + Send + Sync + 'static,
-            #error_path: From<quickwit_actors::AskError<E>>,
+            #error_type: From<quickwit_actors::AskError<E>>,
         {
             type Response = T;
-            type Error = #error_path;
+            type Error = #error_type;
             type Future = BoxFuture<Self::Response, Self::Error>;
 
             fn poll_ready(&mut self, _cx: &mut std::task::Context<'_>) -> std::task::Poll<Result<(), Self::Error>> {
-                //! This does not work with balance middlewares such as `tower::balance::pool::Pool` because this always returns `Poll::Ready`.
-                //! The fix is to acquire a permit from the mailbox in `poll_ready` and consume it in `call`.
+                //! This no longer works with parallel actors. The fix is to acquire a permit from the mailbox in `poll_ready` and consumer it in `call`.
```

Comment:
why did you change the comment? I like the previous one.

Reply:
Rebase gone wrong. I'll fix :)